From 0b19839b5097be84b2948f7e5d96b08de7269a5c Mon Sep 17 00:00:00 2001
From: JoseskVolpe <78318343+JoseskVolpe@users.noreply.github.com>
Date: Thu, 19 Sep 2024 19:36:44 +0000
Subject: [PATCH] hwdb: add keymaps for Acer Nitro 5 AN515-47 (#34493)

This fixes the microphone mute key and NitroSense key for Acer Nitro 5 AN515-47,
so they can be recognized by desktop environments.

(cherry picked from commit 3135a21be422f3c33e445bf249dc5df1a74f9dc0)

Resolves: RHEL-49810
---
 hwdb.d/60-keyboard.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 27be239bc6..56bae850e1 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -235,6 +235,11 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredatorPHN16-71:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredatorPHN16-72:*
  KEYBOARD_KEY_66=micmute                                # Microphone mute button
 
+# Nitro AN515-47
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnNitro*AN*515-47:pvr*
+ KEYBOARD_KEY_66=micmute                                # Microphone mute button
+ KEYBOARD_KEY_f5=prog1                                  # NitroSense button
+
 # Nitro AN515-58
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnNitro*AN*515-58:pvr*
  KEYBOARD_KEY_8a=f20                                    # Microphone mute button
